cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KRaftMigrationUtils.java

```diff
@@ -33,6 +34,8 @@ public class KRaftMigrationUtils {
     /**
      * This method deletes the /controller znode from ZooKeeper to allow the brokers, which are now in ZooKeeper mode again,
      * to elect a new controller among them taking the KRaft controllers out of the picture.
+     * It also deletes the /migration znode from ZooKeeper to reset the migration status in case of rollback and the user
+     * wants to start a KRaft migration again.
```

Comment:
Should we also add why this is needed? I mean to avoid skipping the initial ZK to KRaft sync.

Reply:
Done.